packages/client/src/util/mixpanel.ts

```diff
@@ -34,6 +34,10 @@ class MixpanelWrapper {
         mixpanel.track(...args);
     }
 
+    identify(id: string) {
+        mixpanel.identify(id);
```

Comment:
Can you add a guard here like in `track`? These lines:

```
        // No-op if not using
        if (!this.isActive) return;
```

Reply:
sure, the changes are made